.vscode/launch.json

```diff
@@ -15,6 +15,19 @@
       ],
       "console": "integratedTerminal",
       "internalConsoleOptions": "neverOpen"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Jest Tests in Windows",
+      "program": "${workspaceFolder}/node_modules/.bin/jest",
+      "args": ["--runInBand"],
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "disableOptimisticBPs": true,
+      "windows": {
+        "program": "${workspaceFolder}/node_modules/jest/bin/jest"
+      }
```

Comment:
Can you try adding this `windows` part to the current config and see if you're still getting the error?

Reply:
Unfortunately, only adding windows in the ```Debug Jest Tests``` configuration doesn't solve the issue unless I modify the ```runtimeArgs``` like this:
```json
"args": ["--runInBand"],
"windows": {
  "program": "${workspaceFolder}/node_modules/jest/bin/jest"
}
```

What do you think about using only the first configuration but modifying these two parameters?

Reply:
Yes, I think it'd be preferable to fix the existing config